src/interfaces/jira.ts

```diff
@@ -190,3 +190,9 @@ export interface JiraRemoteLinkStatus {
 // https://atlassian.design/components/lozenge/examples
 export type JiraRemoteLinkStatusAppearance = "default" | "inprogress" | "moved" | "new" | "removed" | "prototype" | "success";
 
+export type JiraOperationType = "NORMAL" | "BACKFILL"
```

Comment:
not much point in exporting the type if it's not being used anywhere else. Could probably just inline it within `JiraSubmitOptions`